python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -205,54 +202,54 @@ class ProcessBrightStarsTask(pipeBase.PipelineTask):
         required to normalize each bright star stamp as their central pixels
         are likely saturated and/or contain ghosts, and cannot be used.
     """
+
     ConfigClass = ProcessBrightStarsConfig
     _DefaultName = "processBrightStars"
 
     def __init__(self, butler=None, initInputs=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # Compute (model) stamp size depending on provided "buffer" value
-        self.modelStampSize = [int(self.config.stampSize[0]*self.config.modelStampBuffer),
-                               int(self.config.stampSize[1]*self.config.modelStampBuffer)]
+        self.modelStampSize = [
+            int(self.config.stampSize[0] * self.config.modelStampBuffer),
+            int(self.config.stampSize[1] * self.config.modelStampBuffer),
+        ]
         # force it to be odd-sized so we have a central pixel
         if not self.modelStampSize[0] % 2:
             self.modelStampSize[0] += 1
         if not self.modelStampSize[1] % 2:
             self.modelStampSize[1] += 1
         # central pixel
-        self.modelCenter = self.modelStampSize[0]//2, self.modelStampSize[1]//2
+        self.modelCenter = self.modelStampSize[0] // 2, self.modelStampSize[1] // 2
         # configure Gaia refcat
         if butler is not None:
-            self.makeSubtask('refObjLoader', butler=butler)
+            self.makeSubtask("refObjLoader", butler=butler)
 
     def applySkyCorr(self, calexp, skyCorr):
         """Apply correction to the sky background level.
 
-        Sky corrections can be generated with the 'skyCorrection.py'
-        executable in pipe_drivers. Because the sky model used by that
-        code extends over the entire focal plane, this can produce
-        better sky subtraction.
-        The calexp is updated in-place.
+        Sky corrections can be generated using the ``SkyCorrectionTask``.
+        As the sky model generated there extends over the full focal plane,
+        this should produce a more optimal sky subtraction solution. The calexp
+        is updated in-place.
```

Comment:
The last sentence should probably be under Notes section to bring to attention that `calexp` is modified.